tests/lib/acpi/src/main.c

```diff
@@ -17,14 +17,39 @@ ZTEST(acpi, test_mcfg_table)
 	zassert_not_null(mcfg, "Failed to get MCFG table");
 }
 
-ZTEST(acpi, test_irq_routing_table)
+ZTEST(acpi, test_dev_enum)
 {
-	static ACPI_PCI_ROUTING_TABLE irq_prt_table[CONFIG_ACPI_MAX_PRT_ENTRY];
-	int status;
+	struct acpi_dev *dev;
+	ACPI_RESOURCE *res_lst;
+	int ret;
 
-	status = acpi_get_irq_routing_table(CONFIG_ACPI_PRT_BUS_NAME,
-					    irq_prt_table, ARRAY_SIZE(irq_prt_table));
-	zassert_ok(status, "Failed to get PRT");
+	dev = acpi_device_get("PNP0A08", "0");
+
+	zassert_not_null(dev, "Failed to get acpi device with given HID");
+
+	ret = acpi_current_resource_get(dev->path, &res_lst);
+
+	zassert_ok(ret, "Failed to get current resource setting");
+}
+
+ZTEST(acpi, test_resource_enum)
+{
+	struct acpi_dev *dev;
+	struct acpi_irq_resource irq_res;
+	struct acpi_mmio_resource mmio_res;
+	int ret;
+
+	dev = acpi_device_get("PNP0B00", "0");
```

Comment:
Same here. In this case I think a DT alias is the right way to go since it's getting more platform specific (seems like this is the HID value for "AT Real-Time Clock" according to ACPICA)

Reply:
updated with test case enable with conditional build. This test case builds only if board rtc node have acpi hid property.